pages/dynamic-portal.tsx

```diff
@@ -532,18 +501,138 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     req: context.req,
   });
 
-  if (!token?.sub) {
+  if (!token) {
     console.log("No session token found");
 
     return {
       notFound: true,
     };
   }
 
-  const accessToken = await getAccessToken();
-  const baseConfig = await getSpaceConfig(accessToken);
-
   const prisma = prismaClient;
+  const environmentToken = process.env.DYNAMIC_TEMPLATES_ENVIRONMENT_ID;
```

Comment:
Can you access a NEXT_PUBLIC_ env var from the backend too? If so might as well make this env ID public and not have to pass it via props, but just reference it directly like the publishable key

Reply:
Nope. NEXT_PUBLIC not allowed in server